src/autogpt_plugins/unzipped_plugins/__init__.py

```diff
@@ -0,0 +1,183 @@
+import os
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple, TypeVar, TypedDict
+from auto_gpt_plugin_template import AutoGPTPluginTemplate
+from .plugin_manager import PluginManager
+
+PromptGenerator = TypeVar("PromptGenerator")
+
+
+class Message(TypedDict):
+    role: str
+    content: str
+
+
+class AutoGPTAllowUnzippedPlugins(AutoGPTPluginTemplate):
+    """
+    This plugin allows developers to use unzipped plugins simplifying the plugin development process.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._name = "Auto-GPT-Allow-Unzipped-Plugins"
+        self._version = "0.2.0"
+        self._description = "This plugin allows developers to use unzipped plugins simplifying the plugin development process."
+
+        from autogpt.config.config import Config
+
+        cfg = Config()
+        plugins_dir = Path(cfg.plugins_dir)
+        if os.getenv("UNZIPPED_PLUGIN_INSTALL_REQUIREMENTS", "True") == "True":
+            PluginManager.install_plugin_requirements(plugins_dir)
+        self._plugins = PluginManager.load_unzipped_plugins(plugins_dir)
```

Comment:
@collijk may not love this?

Reply:
Updated to remove the reliance on global config.